pkg/controller/reconciler/reconciler.go

```diff
@@ -121,6 +133,36 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	return ctrl.Result{}, nil
 }
 
+func (r *Reconciler) addFinalizer(ctx context.Context, res common.PlatformObject) error {
+	if len(r.Finalizer) == 0 {
```

Comment:
Could you elaborate a bit logic behind it?

Reply:
mh, smells like some code that should not be in this pr, let me remove it

Reply:
This is, btw, the one of the reasons why I prefer patchsets with logically separated changes with full description of the intentions in the commit message and without unrelated changes :)